src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -282,47 +355,49 @@ The Che CLI is constructed of multiple Docker images within the Che source repos
 
-This puppet-based approach allow us to simplify the outputs for end users and limit the locations where end users need to configure various parts of the system. One powerful example of this is that we generate two `docker-compose.yml` files from a single Puppet template. In the user’s `/instance` folder is `docker-compose.yml` and `docker-compose-container.yml`. The first one is a configuration file that allows a user to run Docker compose for Che on their host. They can just `docker-compose up` in that folder. The second file is for running Docker compose from within a container, which is what the CLI does. The syntax of Docker compose changes in each of these scenarios as the files being referenced from within the compose syntax are different. In the `init` image, we have a single template for Docker Compose and then apply it in two configurations using Puppet.
+This Puppet-based approach allows us to simplify the outputs for end users and limit the locations where end users need to configure various parts of the system. One powerful example of this is that we generate two `docker-compose.yml` files from a single Puppet template. The `docker-compose.yml` and `docker-compose-container.yml` files are located in the user’s `/instance` directory. The first one is a configuration file that allows a user to run Docker compose for Che on their host. They can run the `docker-compose up` command in that directory. The second file is for running Docker compose from within a container, which is what the CLI does. The syntax of Docker compose changes in each of these scenarios as the files being referenced from within the compose syntax are different. There is a single template for Docker Compose in the `init` image. It is then applied in two configurations using Puppet.
 
 [id="cli-tests"]
-== CLI Tests
+== CLI tests
 
-There are existing https://github.com/sstephenson/bats[bats] tests for the Che CLI, which runs automatically with each execution of a `build.sh` script located in the `dockerfiles/cli` folder. To skip them, pass `--skip-tests` argument when running the build script. If you want to just run tests you can achieve it by running script `test.sh` located in the same folder. Tests utilizes docker image `eclipse/che-bats` which is build from `Dockerfile` placed in `dockerfiles/bats`.
+There are existing https://github.com/sstephenson/bats[bats] tests for the Che CLI, which run automatically with each execution of a `build.sh` script located in the `dockerfiles/cli` directory. To skip them, pass the `--skip-tests` argument when running the build script. If you want to just run tests you can achieve it by running the `test.sh` script located in the same directory. Tests utilizes docker image `eclipse/che-bats` which is build from `Dockerfile` placed in `dockerfiles/bats`.
```

Comment:
The final sentence here is a bit of a mess. I'd consider rewriting it both to be clearer and to be grammatically correct.